matlab-plugin-agent/src/main/resources/+ciplugins/+teamcity/TestVisualizationPlugin.m

```diff
@@ -0,0 +1,27 @@
+classdef TestVisualizationPlugin < matlab.unittest.plugins.TestRunnerPlugin
+
+%   Copyright 2025 The MathWorks, Inc.
+
+    methods (Access=protected)
+
+        function runTest(plugin, pluginData)
+            classAndTestName = strrep(pluginData.Name,"/",".");
+            
+            fprintf("##teamcity[testStarted name='%s' captureStandardOutput='true']", classAndTestName)
+
+            % Invoke super class method to run the test
+            runTest@matlab.unittest.plugins.TestRunnerPlugin(plugin, pluginData);
+
+            result = pluginData.TestResult;
+
+            if result.Failed
+                fprintf("##teamcity[testFailed name='%s' status='ERROR']", classAndTestName);
+            elseif result.Incomplete
+                fprintf("##teamcity[testIgnored name='%s']", classAndTestName);
+            end    
```

Comment:
I was confused for a minute here because Incomplete tests aren't always incomplete because they are skipped/ignored. Sometimes exceptions or assertions can also result in Incomplete status, but it looks like it's all good because you are using elseif and checking for Failed status first so I think the only way to hit the Incomplete block is via an assumption failure, which seems like the correct case for this status.